canal/sync.go

```diff
@@ -22,15 +23,15 @@ func (c *Canal) startSyncer() (*replication.BinlogStreamer, error) {
 		if err != nil {
 			return nil, errors.Errorf("start sync replication at binlog %v error %v", pos, err)
 		}
-		log.Infof("start sync binlog at binlog file %v", pos)
+		log.Debugf("start sync binlog at binlog file %v", pos)
```

Comment:
You might think there is a need to change the log lever for these information messages.

But it should be done in another PR, not this one.

It brings confusion, especially because there are no links between these changes and the topic of the PR